pipeline/transform/matches_test.py

```diff
@@ -4,13 +4,15 @@
     convert_matches_from_df,
     filter_recent_matches,
 )
-from pipeline.types import Match
+from pipeline.types import Match, MatchMetadata
 
 
 def test_convert_matches_from_df():
     df = pd.DataFrame([{"users": {"A", "B"}, "extraData": "ignore"}])
     actual = convert_matches_from_df.run(df)
-    expected = [Match(users={"A", "B"})]
+    expected = [
+        Match(users={"A", "B"}, metadata=MatchMetadata(generator="blank"))
+    ]
```

Comment:
Can you add another test case? This only tests that it works when there is no metadata passed in (which is an important case).

We also need a case where there is metadata in the input dataframe to transform.